contactaccessrights/js/test/controllers/access-rights-modal-ctrl_test.js

```diff
@@ -223,6 +223,31 @@ define([
           });
         });
 
+        describe("when location changes", function () {
+          it("cannot submit without change", function () {
+            expect(ctrl.canSubmit).toBe(false);
+          });
+
+          it("can submit when location and region changes", function () {
```

Comment:
Move the setup of the test case to beforeeach, the it block should contain only the expect statement

```
describe('when location and region changes', function () {
  
  beforeEach(function () {
    ctrl.selectedData.locations.push({
      name: "Home",
      label: "Home"
    });

    ctrl.selectedData.regions.push({
      name: "Home",
      label: "Home"
    });

    ctrl.selection();
  });
  
  it("can submit", function () {
    expect(ctrl.canSubmit).toBe(true);
  });
});
```